diff --git a/lib/generated/idl/Location.js b/lib/generated/idl/Location.js
index 66604fab..5249a1c9 100644
--- a/lib/generated/idl/Location.js
+++ b/lib/generated/idl/Location.js
@@ -333,7 +333,7 @@ function getUnforgeables(globalObject) {
       hostname: { configurable: false },
       port: { configurable: false },
       pathname: { configurable: false },
-      search: { configurable: false },
+      search: { configurable: true },
       hash: { configurable: false }
     });
     unforgeablesMap.set(globalObject, unforgeables);
